### ISSUE TITLE
Publish to CRAN

### ISSUE THREAD

**Issue by thomasp85** [dickoa/cgalh#1]
Is there any timeframe for a CRAN release? I hope to soon begin to think about getting euclid on CRAN

**Comment by dickoa** [comment]
Hi @thomasp85 
I can submit this Monday, hopefully it won't take much to have it on CRAN.
Looking forward to seeing euclid on CRAN soon.
Thanks

**Comment by thomasp85** [comment]
gentle bump :-)

**Comment by dickoa** [comment]
Thanks for the reminder @thomasp85. I'll push it!

**Comment by thomasp85** [comment]
what is the target date? (no pressure, just want to know a probable timeline)

**Comment by dickoa** [comment]
Hi @thomasp85,

Just submitted it!

There are two issues in the cran check, the length of some paths exceeding the default length on Windows and the whole size of the headers (>40mb).

Thanks again

**Comment by thomasp85** [comment]
Lovely - thanks. Are these issues new compared to cgal4h?

**Comment by dickoa** [comment]
No, they were already there in cgal4h but I remember not porting the part of the header files with long paths (a dozen). So the only issue was the size of the header files too large for cran standard. This time, I want to maintain the headers without altering them, so I'll try to explain to them why it's important to keep it as it is.

**Comment by thomasp85** [comment]
I see - I hope they will listen

**Comment by thomasp85** [comment]
Don't know if it provides leverage but you are welcome to mention my work as a motivating factor

**Comment by thomasp85** [comment]
is it still in CRAN limbo?

**Comment by dickoa** [comment]
Hi @thomasp85 
I sent two follow up to Uwe but no response. I'll try to re-submit and send one again today.
This is really weird.

**Comment by m-jahn** [comment]
I am very much interested in this too. I want to submit my package https://github.com/m-jahn/WeightedTreemaps to CRAN (where you co-authored some lines of code), but always run into compiler issues on the pedantic compilers with sanitizing (ASAN/UBSAN). I get loads of errors from upstream dependencies boost (BH) and CGAL (your package cgal4h), and I am assuming this is due to deprecated functions. Hope with newest CGAL and boost libraries these issues can be resolved. You can give me mention as soon as the package is on CRAN.
